fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -2157,4 +2159,17 @@ public class Config extends ConfigBase {
             "是否禁止LocalDeployManager删除节点",
             "Whether to disable LocalDeployManager drop node"})
     public static boolean disable_local_deploy_manager_drop_node = true;
+
+    @ConfField
+    public static long huge_table_auto_analyze_interval_in_millis = TimeUnit.HOURS.toMillis(12);
+
+    @ConfField
+    public static long huge_table_lower_bound_size_in_bytes = 5L * 1024 * 1024 * 1024;
```

Comment:
I think user may not know how to set this config.

Reply:
Yes, but I think we still should provide a way to make sure DBA could interfere with its behavior when necessary